MapleServer2/Tools/SockHint.cs

```diff
@@ -29,11 +29,39 @@ public struct SockHintInfo
 
         public SockHintInfo(SockHint hint, string value, string name = "Unknown")
         {
-            this.Hint = hint;
-            byte.TryParse(value, out ByteValue);
-            short.TryParse(value, out ShortValue);
-            int.TryParse(value, out IntValue);
-            long.TryParse(value, out LongValue);
+            Hint = hint;
```

Comment:
Dotnet rules require TryParse to be implemented in this way, I think that without this implementation it would technically error if the value was unable to be parsed.

Reply:
I see. I do feel that this could be coded in a much more eloquent way, but this is fine for now I suppose.

Reply:
I cleaned up the refactor a bit, should be good now.